Modules/_ctypes/cfield.c

```diff
@@ -71,6 +71,18 @@ PyCField_FromDesc(PyObject *desc, Py_ssize_t index,
         Py_DECREF(self);
         return NULL;
     }
+
+#ifndef MS_WIN32
+    if (pack && bitsize) { /* packed bitfield */
+        size = 1;
+        while(size * 8 < bitsize)
+            size += 1;
```

Comment:
If we have packed bitfield, calculate the minimum number of bytes needed to fit the bitfield.